laser_encoders/README.md

```diff
@@ -25,10 +25,21 @@ You can install laser_encoders using pip:
 
 ## Usage
 
-Here's a simple example of how you can download and initialise the tokenizer and encoder with just one step.
+Here's a simple example on how to obtain embeddings for sentences using the `LaserEncoderPipeline`:
 
-**Note:** By default, the models will be downloaded to the `~/.cache/laser_encoders` directory. To specify a different download location, you can provide the argument `model_dir=path/to/model/directory` to the initialize_tokenizer and initialize_encoder functions
+>**Note:** By default, the models will be downloaded to the `~/.cache/laser_encoders` directory. To specify a different download location, you can provide the argument `model_dir=path/to/model/directory`
 
+```py
+from laser_encoders import LaserEncoderPipeline
```

Comment:
Can you please add a unit test covering this case?
And by the way make sure that all the other examples in the readme are covered by tests

Reply:
Alright, Will do that.

Reply:
Hi @avidale @heffernankevin ,
I have just added tests to the PR.

I am seeking some clarification regarding a couple of things:

1. Should I also include tests for the "Downloading the pre-trained models" section of the readme? It involves running some commands in the terminal and using `SentenceEncoder` and `LaserTokenizer`, which I believe are already covered by the current tests. Please let me know your thoughts.

2. Was it an intentional decision to use a temporary directory (`tmp_path`) for the tests? It seems like the same files may be downloaded for every test, which extends the time it takes to run them.

Thank you!

Reply:
1. No, I don't think that testing the downloading is obligatory, if it is already included in other tests.
2. Yes, it was an intentional decision; generally, it makes sense to run each test in a clean environment. 
However, if you feel that this significantly slows down your development process, we can think of a better way to run the tests. 
For example, we can bundle the tests into one class, and use [class-level setup and teardown methods](https://docs.pytest.org/en/latest/how-to/xunit_setup.html#class-level-setup-teardown) for creating and destroying the cache directory. I don't think it is a must, but you can do this, if it helps you.